src/aliceVision/sfmDataIO/plyIO.cpp

```diff
@@ -53,36 +60,61 @@ bool savePLY(
       << '\n' << "property uchar blue"
       << '\n' << "end_header" << std::endl;
 
-      if (b_extrinsics)
-      {
-        for (const auto& view : sfmData.getViews())
+        if (b_extrinsics)
```

Comment:
I know this is not part of your PR. However, do you need the camera position to be stored ?

Reply:
De hozzátettem a kompatibilitás miatt